From: Dominik George <natureshadow@debian.org>
Summary: Fixes make install due to duplicate file name in /usr/bin/install call
Index: pg-rrule/Makefile
===================================================================
--- pg-rrule.orig/Makefile
+++ pg-rrule/Makefile
@@ -21,7 +21,7 @@ all: sql/$(EXTENSION)--$(EXTVERSION).sql
 sql/$(EXTENSION)--$(EXTVERSION).sql: sql/$(EXTENSION).sql
 	cp $< $@
 
-DATA = $(wildcard sql/*--*.sql) sql/$(EXTENSION)--$(EXTVERSION).sql
+DATA = sql/$(EXTENSION)--$(EXTVERSION).sql
 EXTRA_CLEAN = sql/$(EXTENSION)--$(EXTVERSION).sql
 endif
 
